azure-devops/golang/stages/40-delivery/arm.yaml

```diff
@@ -71,4 +72,16 @@ stages:
               else
                 echo "No Goose migration files found. Skipping Goose installation and migration."
               fi
+          - script:
+              aaz postgres flexible-server firewall-rule delete \
```

Comment:
I think `aaz` was a typo right?

Reply:
Yes, it was. I'm sending the correction.